src/cluster.c

```diff
@@ -4390,6 +4476,80 @@ sds clusterGenNodesDescription(int filter, int use_pport) {
     return ci;
 }
 
+/* Add to the output buffer of the given client the description of the given cluster link.
+ * The description is a map with each entry being an attribute of the link. */
+void addReplyClusterLinkDescription(client *c, clusterLink *link) {
+    addReplyMapLen(c, 7);
+
+    addReplyBulkCString(c, "fd");
+    addReplyLongLong(c, link->conn->fd);
```

Comment:
is it really needed to expose the fd to users?

Reply:
For some reason I thought we also exposed the FD in client list, but we don't. I would agree that I don't think we should emit it.

Reply:
Will remove.